### PR TITLE
Validation front

### PR DESCRIPTION
**Pull request by maltaesousa**
This PR finalizes validation functionality for layers that need validation before export. IE:
1. at700 needs validation by Mr. D
2. admin creates metadata for at700 with 2 contact persons, on of them is a validator
3. Client orders PCAZ group in which at700 is present
4. Mr D. get an email asking for validation, he can cancel or deny
5. Meanwhile, other products are processed
6. When everything is extracted, client gets the final email

Mail preview:
image

Validation preview:
image

Tests passing, deployed on prepub but take care testing because real emails will be sent.

### CODE REVIEW

File: back/api/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -72,485 +72,523 @@ msgstr "Recherche"
...
-#: api/models.py:147
+#: api/models.py:150
+#, fuzzy
+#| msgid "is_public"
+msgid "Public"
+msgstr "Public"
+
+#: api/models.py:151
+msgid "Approval needed"
+msgstr "Diffusion restreinte"
```
Comment (kalbermattenm) [inline back/api/locale/fr/LC_MESSAGES/django.po:204]: Is that the right translation?

File: back/api/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -72,485 +72,523 @@ msgstr "Recherche"
...
 
-#: api/models.py:150
+#: api/models.py:157
 msgid "datasource"
 msgstr "source des données"
 
-#: api/models.py:151
+#: api/models.py:159
+msgid "accessibility"
+msgstr "Accèssibilité"
```
Comment (kalbermattenm) [inline back/api/locale/fr/LC_MESSAGES/django.po:224]:
```suggestion
msgstr "Accessibilité"
```

File: front/src/app/welcome/validate/validate.component.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    this.route.params
+      .pipe(takeUntil(this.onDestroy$))
+      .subscribe(params => this.token = params.token);
+  }
+
+  ngOnInit(): void {
+    this.apiOrderService.getOrderItemByToken(this.token)
+      .subscribe(iOrderItem => {
+        if (iOrderItem) {
+          console.log(iOrderItem)
```
Comment (kalbermattenm) [inline front/src/app/welcome/validate/validate.component.ts:52]: I do not remember if you keep these or not (there's another one on line 57)

**Review by kalbermattenm — APPROVED** [review]
This one is quite hard for me to review, but code seems OK

File: back/api/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (maltaesousa) [reply back/api/locale/fr/LC_MESSAGES/django.po:204]: Yep, how would you say it for a data that needs to be validated by the data owner before it's processed?

File: front/src/app/welcome/validate/validate.component.ts (hunk 3 above)
Comment (maltaesousa) [reply front/src/app/welcome/validate/validate.component.ts:52]: Thanks, I should get rid of that!

**Pull request merged**